### DISCUSSION TITLE
Where to report docs issues ?

### DISCUSSION THREAD

**Discussion by gempain** [GrapesJS/grapesjs#5740] [Q&A]
Hi 👋

Editor event `load` is reported [here](https://github.com/GrapesJS/grapesjs/issues/171#issuecomment-316371318) but [not documented](https://grapesjs.com/docs/api/editor.html#available-events).

Where should I report or PR this ?

Best,

**Comment by artf** [comment]
What do you mean?
<img width="737" alt="Screenshot 2024-03-12 at 23 51 09" src="https://github.com/GrapesJS/grapesjs/assets/11614725/2446af3a-1d11-4d53-8d82-2c37100f85d3">

**Reply by gempain** [reply]
I guess I must have missed it. We looked at it with colleagues and at least 2 of us didn't find it the other day. Strange world.

Anyhow, problem solved then !

It doesn't answer my question though. Where should we PR things we find in the docs that may be incorrect?

**Reply by artf** [reply]
You can submit them to this repo.

Here you can find docs to Modules https://github.com/GrapesJS/grapesjs/tree/dev/docs/modules
But the [API Reference](https://grapesjs.com/docs/api/) docs are generated from the source code.
